phys/module_physics_init.F

```diff
@@ -1517,6 +1525,11 @@ SUBROUTINE phy_init ( id, config_flags, DT, restart, zfull, zhalf,     &
                 adv_moist_cond,                                 &
                 MPDT, DT, DX, DY, LOWLYR,                       &
                 F_ICE_PHY,F_RAIN_PHY,F_RIMEF_PHY,               &
+!-------------------for ntu3m------------------------------------
```

Comment:
@tzuchin12 
Most developers put a comment on a single line after the "&" character. So remove the 
```
!-------------------for ntu3m--
```
where you do not see others doing this.

Reply:
Modified as suggested.